src/payment/offsite-payment-initializer.js

```diff
@@ -58,6 +59,6 @@ export default class OffsitePaymentInitializer {
         const payload = this.payloadMapper.mapToPayload(data);
         const url = this.urlHelper.getOffsitePaymentUrl();
 
-        this.formPoster.postForm(url, payload, callback);
+        this.formPoster.postForm(url, payload, callback, target);
```

Comment:
This depends on https://github.com/bigcommerce/form-poster-js/releases/tag/1.4.0, could you bump the dependency? Also for future reference, it would help us if we didn't just list sibling PRs, but also the order of dependencies. Thanks!

Reply:
Hi @TomA-R, thanks for your feedback. I've updated the description and also the version. Let me know if I did it well.